### PR TITLE
cube browser

### PR DESCRIPTION
**Pull request by marqh**

### CODE REVIEW

**Comment by marqh** [comment]
i don't understand why I had to pin to python2 to get python2 on this PR, I thought this would not be required

this is now only building for python2

comments and suggestions on how to get it to build for both are most welcome

**Comment by QuLogic** [comment]
On that second last build, it was installed to Python 3.5, and the import test was run on 2.7; what?

File: cube_browser/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        - iris_sample_data
+        - six
+        - notebook
+
+test:
+    imports:
+        - cube_browser
+
+about:
+    home: http://scitools.org.uk/cube_browser
```
Comment (QuLogic) [inline cube_browser/meta.yaml]: This link is 404.

**Comment by marqh** [comment]
> On that second last build, it was installed to Python 3.5, and the import test was run on 2.7; what?

indeed, I can't fathom what is taking place here

**Comment by marqh** [comment]
@bjlittle are you in a position to action this?
is there more to do here? (there are rough edges, are they showstoppers?)
this is a blocker at present for other activities

**Comment by marqh** [comment]
rebased following #231

**Pull request closed without merging**